manifests/init.pp

```diff
@@ -9,6 +9,11 @@
   $tools            = true,
 ) inherits pxe::params {
 
+  # Ensure wget is installed before any execs of wget
+  Package <| |> -> Exec <| |>
```

Comment:
This will cause all sorts of problems for users.  You're saying *all* packages have to become before *all* Execs across the entire catalog.

Reply:
Yep. Saw that in @vchepkov 's comment yesterday. Will be fixing.